app/containers/pageListAsync.js

```diff
@@ -1,41 +1,83 @@
-import React from 'react';
-import Loading from 'dan-components/Loading';
-import loadable from '../utils/loadable';
+import React from "react";
+import Loading from "dan-components/Loading";
+import loadable from "../utils/loadable";
 
-export const SuperHome = loadable(() => import('./MassEnergizeSuperAdmin/Pages/Home'), {
-  fallback: <Loading />,
-});
-export const Impact = loadable(() => import('./MassEnergizeSuperAdmin/Pages/Impact'), {
-  fallback: <Loading />,
-});
-export const ImpactPage = loadable(() => import('./MassEnergizeSuperAdmin/Pages/ImpactPage'), {
-  fallback: <Loading />,
-});
-export const SuperAllActions = loadable(() => import('./MassEnergizeSuperAdmin/Pages/AllActions'), {
-  fallback: <Loading />,
-});
+export const AddToGallery = loadable(
```

Comment:
@frimpongopoku - I don't like it when your auto-formatter does stuff like this.  It makes it difficult to review changes, and often makes the code less readable in my opinion.  Can you please turn it off?

Reply:
Hi @BradHN1, I will try my best to check and "unformat" things before I make PRs. Just like you, I am also unable to read and understand the code properly when  it isnt arranged. So I will just try to "unformat" when I am done to help you as well. 
Nevertheless, I do not think we should be fighting against formatters 🤣 🤣 . We cant always be aligning brackets and tracking indentations ourselves when we can just offset it to a formatter; plus, a formatter does it faster, and does it the way its meant to be done. It just takes some getting used to 🤗 🤣 .